manager/src/main/resources/flow-crd.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: flows.streamzi.io
+spec:
+  group: streamzi.io
+  version: v1
+  scope: Namespaced
+  names:
+    plural: flows
+    singular: flow
+    kind: Flow
+    shortNames:
```

Comment:
should we add some 
```
validation:
    openAPIV3Schema:
``` 

for defining the actual structure ?

Reply:
I think we can do this later and re-visit as part of the defining flow structure task.

Reply:
+1